docbook5-book/xml/common_license_gfdl1.2.xml

```diff
@@ -16,6 +16,12 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>GNU Free Documentation License</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager"></dm:docmanager>
+  <dm:bugtracker></dm:bugtracker>
+  <dm:translation>no</dm:translation>
```

Comment:
This results in
```
Fatal error:
/home/cwickert/git/github.com/suse/doc-public-cloud/xml/common_license_gfdl1.2.xml:
    ERROR: Namespace prefix dm on bugtracker is not defined (line 21 column 17)
/home/cwickert/git/github.com/suse/doc-public-cloud/xml/common_license_gfdl1.2.xml:
    ERROR: Namespace prefix dm on translation is not defined (line 22 column 18)
```
The `dm:translation` and `dm:bugtracker` tags need to be part of `dm:docmanager`. Fixed in d8bd3cac6ed923b39b1ca88fde27768583bb507e

Reply:
Argh. The end tag `</dm:docmanager> has to be moved to the end of line 22.